README.md

```diff
@@ -1,12 +1,16 @@
-# Fulfilment House Registration Scheme Application
+# fh-registration-frontend
 
-[![Build Status](https://travis-ci.org/hmrc/fhdds-frontend.svg)](https://travis-ci.org/hmrc/fhdds-frontend) [ ![Download](https://api.bintray.com/packages/hmrc/releases/fhdds-frontend/images/download.svg) ](https://bintray.com/hmrc/releases/fhdds-frontend/_latestVersion)
+[![Build Status](https://travis-ci.org/hmrc/fh-registration-frontend.svg?branch=master)](https://travis-ci.org/hmrc/fh-registration-frontend) [ ![Download](https://api.bintray.com/packages/hmrc/releases/fh-registration-frontend/images/download.svg) ](https://bintray.com/hmrc/releases/fh-registration-frontend/_latestVersion)
 
 This service provides the frontend endpoint for the [Fulfilment House Registration Scheme](https://www.gov.uk/guidance/fulfilment-house-due-diligence-scheme) project.
 
 ## Summary
 
-This service allow a customer to apply for apply for the Fulfilment House Registration Scheme.
+This service allow a customer to apply for the Fulfilment House Registration Scheme.
+
+##Authentication
```

Comment:
Should be a space between the # and Authentication otherwise the mark up will not work